tutorials/5 All QRenderers/5.1 Ansys/Ansys QRenderer.ipynb

```diff
@@ -730,7 +776,7 @@
    "name": "python",
    "nbconvert_exporter": "python",
    "pygments_lexer": "ipython3",
-   "version": "3.7.3"
+   "version": "3.7.8"
```

Comment:
At the end, add a cell  to  close the gui

Suggest:

# If you want to close the Metal GUI
gui.main_window.close()

Reply:
we shuold discuss this thing of the gui closing. I have found it pretty awkward to run cell and at the end get a pop-up "do you really want to close", to which I systematically say no.

I can add the cell but I will comment it out.